eureka/src/test/java/com/linecorp/armeria/client/eureka/EurekaEndpointGroupTest.java

```diff
@@ -29,19 +29,27 @@
 import com.netflix.discovery.converters.wrappers.CodecWrappers;
 import com.netflix.discovery.converters.wrappers.CodecWrappers.JacksonJson;
 import com.netflix.discovery.converters.wrappers.EncoderWrapper;
+import com.netflix.discovery.shared.Application;
 import com.netflix.discovery.shared.Applications;
 import com.netflix.discovery.util.InstanceInfoGenerator;
 
 import com.linecorp.armeria.client.Endpoint;
 import com.linecorp.armeria.common.HttpResponse;
 import com.linecorp.armeria.common.HttpStatus;
 import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.internal.common.eureka.InstanceInfo;
 import com.linecorp.armeria.server.ServerBuilder;
 import com.linecorp.armeria.testing.junit5.server.ServerExtension;
 
+import io.netty.util.AttributeKey;
+
 class EurekaEndpointGroupTest {
 
     private static final EncoderWrapper encoder = CodecWrappers.getEncoder(JacksonJson.class);
+    private static final String APP_WITH_METADATA = "with-metadata";
+    private static final AttributeKey<InstanceInfo> INSTANCE_INFO = AttributeKey.valueOf(
```

Comment:
Let's use https://github.com/line/armeria/pull/3980/files#diff-7bf2b0d8d123c011e14425573a7aad2a7e9a9847fd188d63cbba79d58f7969a9R96
instead of defining a new one.

Reply:
👌  It is updated now.